recipes/flex/all/conanfile.py

```diff
@@ -71,7 +71,10 @@ def package(self):
             autotools = self._configure_autotools()
             autotools.install()
         tools.rmdir(os.path.join(self.package_folder, "share"))
-        os.unlink(os.path.join(self.package_folder, "lib", "libfl.la"))
+        for la in ["libfl.la", "libfl_pic.la"]:
+            la_path = os.path.join(self.package_folder, "lib", la)
+            if os.path.exists(la_path):
+                os.unlink(la_path)
```

Comment:
You can replace these with:
```suggestion
        tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.la")
```
and put `required_conan_version = ">=1.29"` at the top of the file.

Reply:
Done